### ISSUE TITLE
A few 404s on docs site

### ISSUE THREAD

**Issue by pdehaan** [mozilla-lockbox/lockbox-extension#412]
With #411 fixed/merged, the list of 404s shrinks considerably. Not sure what of these are worth fixing. My gut says "all", but I don't feel too passionate about /feed.xml and I don't know how much we care about stuff like SEO/RSS right meow.

<img width="1440" alt="lockox-404s" src="https://user-images.githubusercontent.com/557895/33958161-83940ff8-dff8-11e7-8c76-f1c1cf7592e4.png">

**Comment by devinreams** [comment]
- the `projects` contributing link should and will be fixed
- `feed.xml` can be removed by overriding the template header
- "Edit on GitHub" link for `api.md` is baked into the template and since we programmatically generating the api.md file, I don't think we do anything with this ⚠️

**Comment by pdehaan** [comment]
Current status:

<img width="1440" alt="integrity_v6" src="https://user-images.githubusercontent.com/557895/34022611-18d679b4-e0f5-11e7-9ae3-dc8412a1fef6.png">

Submitted upstream PRs: https://github.com/mozilla-lockbox/lockbox-datastore/pull/66 and https://github.com/mozilla-lockbox/lockbox-ios/pull/83.

I can't think of a good workaround for the broken GitHub generated `docs/api.md` link, unless we commit a dummy api.md file that says:

> ### DO NOT EDIT!
> This file is automatically generated using the <kbd>npm run doc</kbd> script

... and then commit that file to GitHub. Users would still potentially click the "Edit on GitHub" button and go to a somewhat _less_ dead end, but at least it wouldn't 404.

**Comment by devinreams** [comment]
>I can't think of a good workaround for the broken GitHub generated docs/api.md link, unless we commit a dummy api.md file that says:

Good point. The `doc` script should overwrite the dummy page in the process of generating contents so that seems OK to check-in. I like it. @linuxwolf @jimporter any opposition to that?
